### PR TITLE
Correct a few typos in the docs 

### PR DESCRIPTION
**Pull request by jenshnielsen**
Numpy to Scipy (Matches the comments in the code and I can find the code in Scipy)

128 bit reals to long double which is what we actually support

### CODE REVIEW

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
retest this please

**Comment by jenshnielsen** [comment]
retest this please

File: documentation/source/api/python.rst [hunk 1]
```diff
@@ -28,14 +28,16 @@ The latter are first converted to a specific numpy type, depending on how this p
...
+doubles are used through out. The exact size of long doubles is platform dependent
+and is typically 80-bit on Linux and OSX and 64-bit on Windows. See `Wikipedia
+<https://en.wikipedia.org/wiki/Long_double>`__ for more information.
+Otherwise, the package defaults to garden variety 64-bit doubles.
 
 
 Linear Algebra
 ++++++++++++++
 
-Numpy does provides only a subset of its utilities for arrays consisting of 128 bit reals. As a
```
Comment (garymacindoe) [inline documentation/source/api/python.rst]: Should either be 'Numpy provides only' or 'Numpy does provide only'.

**Comment by jenshnielsen** [comment]
fixed

**Pull request merged**